.phpcs.xml.dist

```diff
@@ -12,18 +9,26 @@
 
 	<rule ref="WordPress.WP.I18n">
 		<properties>
-			<property name="text_domain" type="array" value="jetpack" />
+			<property name="text_domain" type="array">
+				<element value="jetpack">
```

Comment:
As a single element, this needs a closing `/>`.

Reply:
b423a9afe